src/Subsystems/Chassis.h

```diff
@@ -20,17 +24,30 @@ class Chassis: public frc::Subsystem {
 
 public:
 
+
+
 	enum ShifterValue{kShifterHigh = DoubleSolenoid::kForward, kShifterLow = DoubleSolenoid::kReverse};
+	enum AutoShifterValue{kAutoShiftOn = true, kAutoShiftOff = false};
```

Comment:
Is `AutoShifterValue` used anywhere?

I like the idea of an enum instead of using bool directly. (There's several methods in CANTalon that would be more understandable with an enum.) So, I would encourage you to use these values.

Not that you don't need to change the type from bool in order to use these. The compiler will happily convert these values to bools for you. 😄